Source/Core/VideoCommon/VertexManagerBase.cpp

```diff
@@ -515,15 +515,23 @@ void VertexManagerBase::Flush()
     auto& counts =
         is_perspective ? m_flush_statistics.perspective : m_flush_statistics.orthographic;
 
+    const auto& projection = xfmem.projection.rawProjection;
     // TODO: Potentially the viewport size could be used as weight for the flush count average.
     // This way a small minimap would have less effect than a fullscreen projection.
+    const auto& viewport = xfmem.viewport;
 
-    if (IsAnamorphicProjection(xfmem.projection.rawProjection, xfmem.viewport, g_ActiveConfig))
+    const float projection_ar = projection[2] / projection[0];
+    const float viewport_ar = viewport.wd / viewport.ht;
+    // FYI: This average is based on flushes.
+    // It doesn't look at vertex counts like the heuristic does.
+    counts.average_ratio.Push(std::abs(projection_ar / viewport_ar));
```

Comment:
This duplicates the CalculateProjectionViewportRatio function (minus the Push).

Reply:
fixed.